polyglot/piranha/demo/find_replace_custom_cleanup_demos.py

```diff
@@ -14,5 +14,16 @@ def java_demo():
     print("Running the Find/Replace Custom Cleanup demo for Java")
     _ = run_piranha_cli(join(find_Replace_dir, "java"), join(find_Replace_dir, "java/configurations"), True)
 
+def python_demo():
```

Comment:
Can we please add another small rule that updates "dependency3" -> "dependency3_1"